gtwrap/interface_parser/function.py

```diff
@@ -47,7 +47,7 @@ def __init__(self,
             self.ctype = ctype
         self.name = name
         self.default = default
-        self.parent: Union[ArgumentList, None] = None
+        self.parent = None  # type: Union[ArgumentList, None]
```

Comment:
Thanks Varun. A few questions:

1. We are removing the first type annotation. But there are 10 more below. So is it really a fix?
2. I can't for the life of me figure out where Python 2 is being used in the wheel builder. I assume any Python 3 can support type hints. I would prefer to just upgrade the python version. Python2 is dead to me.

Reply:
1. Python 3.5 doesn't support variable annotations but it supports function annotations. More [here](https://www.python.org/dev/peps/pep-0526/).
2. It is being set in the `.github` directory. We don't support Python2 here so no need to worry about that. :slightly_smiling_face:

Reply:
Thanks Varun. Do we still need support for Python 3.5?

Reply:
We should until there is a strong reason not to? Some organizations may not have upgraded.